.ci/Makefile

```diff
@@ -37,10 +37,12 @@ env:
 
 # Create Docker image
 build-docker: validate-version
+	@echo "Obsoleted, use GitHub actions"
 	docker build -t $(DOCKER_REGISTRY)/$(DOCKER_IMAGE_NAME):$(AGENT_VERSION) \
   --build-arg AGENT_DIR=$(DIST_DIR) ..
 
 push-docker: build-docker
+	@echo "Obsoleted, use GitHub actions"
```

Comment:
Moving away from the push-docker.sh script drops support for only *sometimes* applying the "latest" Docker tag to an image. This handling was used so that we could use the same push-docker.sh script for the current latest major releases (4.x) and for the 3.x maintenance branch.

I suppose this is fine, given that we won't add provenance support to the 3.x branch.  At some point if/when tehre is a 5.x major, then we'd need to make sure that a maintenance 4.x release does *NOT* push the "latest" Docker tag.

Reply:
Shall we move this topic to a follow-up?

Reply:
Yes, this PR doesn't need to handle this.